superset-frontend/src/components/UserStack/index.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { getCategoricalSchemeRegistry } from '@superset-ui/core';
+import { Avatar, Tooltip } from 'src/common/components';
+import { getRandomColor } from './utils';
+
+interface UserStackProps {
+  users: { first_name: string; last_name: string }[];
+  maxCount?: number;
+}
+
+const colorList = getCategoricalSchemeRegistry().get()?.colors ?? [];
+const AVATAR_STYLE = {
+  width: '24px',
+  height: '24px',
+  fontSize: '20px',
+  lineHeight: '24px',
+};
+
+export default function UserStack({ users, maxCount = 4 }: UserStackProps) {
+  return (
+    <Avatar.Group maxCount={maxCount} maxStyle={AVATAR_STYLE}>
+      {users.map(({ first_name, last_name }) => {
+        const name = `${first_name} ${last_name}`;
+        return (
+          <Tooltip key={name} title={name} placement="top">
```

Comment:
this `key` won't be unique if the same user is an owner of multiple dashboards i think. Perhaps we need to append more context here?

Reply:
this is rendered in the context of a single row for dashboards. The array here is for owers on a single dashboard and that list should be unique. While the key would not be unique for the whole page, it would be unique for the list in which each appears.